go/libraries/doltcore/doltdb/doltdb.go

```diff
@@ -187,8 +187,8 @@ func (ddb *DoltDB) WriteEmptyRepoWithCommitTime(ctx context.Context, name, email
 	return err
 }
 
-func getCommitStForRef(ctx context.Context, db datas.Database, dref ref.DoltRef) (types.Struct, error) {
-	ds, err := db.GetDataset(ctx, dref.String())
+func getCommitStForDatasetName(ctx context.Context, db datas.Database, datasetName string) (types.Struct, error) {
```

Comment:
Typically don't let the term "dataset" bleed into dolt code as it requires devs to understand what a dataset is in noms.

Reply:
Is `getCommitStForRef` better here, even if it doesn't take a `ref.DoltRef`?